### PR TITLE
1 0.8.7.1 Preflight Core Tests

### PR DESCRIPTION
**Pull request by wanda-carlson**
Simple summary [Notion](https://www.notion.so)

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Screen Shot 2022-01-06 at 7 46 07 AM

Attached an image for preview

**Comment by wanda-carlson** [comment]
**Henry Richard** what do you think?

**Comment by henry-richard** [comment]
This looks great!

File: src/bert.py [hunk 1]
```diff
@@ -12,3 +12,4 @@
 # 0.8.7 Preflight 1
 # 2 0.8.7 Preflight
```
Comment (henry-richard) [inline src/bert.py:13]: Remove these

File: src/bert.py [hunk 2]
```diff
@@ -12,3 +12,4 @@
...
 # 1 0.8.7.1 Preflight
+# 1 0.8.7.1 Preflight Core Tests
```
Comment (henry-richard) [inline src/bert.py:15]: And this!

**Review by henry-richard — CHANGES_REQUESTED** [review]
minor changes

**Review by henry-richard — CHANGES_REQUESTED** [review]
make more changes

**Review by henry-richard — APPROVED** [review]

**Pull request merged**